src/sentry/models/release.py

```diff
@@ -207,9 +212,10 @@ def filter_by_semver(
                 function="ROW",
             )
             cols = self.model.SEMVER_COLS[: len(semver_filter.version_parts)]
-            qs = qs.annotate(
+            q = qs.annotate(
```

Comment:
Nit: I'd probably still call this `qs` since `.annotate` will return a `Queryset` rather than a `Q`